perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -359,11 +360,11 @@ def Prepare(benchmark_spec):
   # Choose a disk or file name and optionally fill it
   disk = vm.scratch_disks[0]
 
-  if FLAGS.against_device or FLAGS.prefill_device:
+  if FLAGS.fio_target_mode in FILL_TARGET_MODES:
     logging.info('Fill device %s on %s', disk.GetDevicePath(), vm)
-    FillDevice(vm, disk, FLAGS.device_fill_size)
+    FillDevice(vm, disk, FLAGS.fio_fill_size)
 
-  if not FLAGS.against_device and FLAGS.prefill_device:
+  if FLAGS.fio_target_mode == 'against_file_with_fill':
```

Comment:
Put a comment here to clarify why we only need to mount the FS here in the against_file_with_fill mode, because:

1) if it's in any of those two against_device modes, we should not mount FS.
2) if it's in against_file_without_fill mode, we should not need to mount the FS here, because the FS was never unmounted in this mode - see the code in GetConfig() which does not clears the mount point when we are in against_file_without_fill mode.

It took me a bit of time to read this code, adding a comment like this helps future readers to figure this out quickly, saving their time.